### ISSUE TITLE
Patch contact

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#466]
I do the following requests that should be valid. Mostly is due to missing type for im/email/phone..

## Patch emails:

```
{
	"current_state": {
		"emails": [{
			"email_id": "f6d5d142-9fb7-494a-80a6-4b39bc199c24",
			"is_primary": false,
			"label": "",
			"type": "other",
			"address": "acm@name.tld"
		}]
	},
	"emails": [{
		"email_id": "f6d5d142-9fb7-494a-80a6-4b39bc199c24",
		"is_primary": false,
		"label": "",
		"type": "other",
		"address": "acm@muc.de"
	}, {
		"address": "foo@bar.tld"
	}]
}
```

first response is 204 but when trying to patch again I get 409 and 500 when I fetch contacts.
And when I fetch the contact, the `emails` property has gone.

## Patch phone

without type:

```
{"current_state":{},"phones":[{"number":"+33612364578"}]}
```

Patch 204 but next GET is 304 and next patch 409

## Patch im

fail same way

## Patch address

```
{"current_state":{},"addresses":[{}]}
{"current_state":{},"addresses":[{"street":"45"}]}
{"current_state":{},"addresses":[{"postal_code":"65454"}]}
```

-> err 400

```
{"current_state":{},"addresses":[{"city":"azdazd"}]}
{"current_state":{},"addresses":[{"street":"rue","city":"ville"}]}
{"current_state":{},"addresses":[{"street":"rue","city":"ville","postal_code":"123456"}]}
```

-> status 204 but next GET 304 and next patch 409

**Comment by sapiens-sapide** [comment]
Need to be reproduced by someone else because « cmcm » (it works for me) !

**Comment by iamdey** [comment]
strange. May be the version of cassandra or something related to environment

**Comment by sapiens-sapide** [comment]
Fix in PR #467
